manifests/install.pp

```diff
@@ -55,5 +58,13 @@
       tag    => 'nodesource_repo',
     }
   }
-}
 
+  file { 'root_npmrc':
```

Comment:
I did this because the docs read that this file is managed, even though it actually is not. So this is a step to get reality to match the docs.

I don't have any strong feelings between managing the contents of the file as it is now or using file_line.